gitlab/tests/common.py

```diff
@@ -290,4 +290,7 @@ def assert_check(aggregator, metrics, use_openmetrics=False):
         )
 
     for metric in metrics:
-        aggregator.assert_metric("gitlab.{}".format(metric))
+        aggregator.assert_metric("gitlab.{}".format(metric), at_least=0)
```

Comment:
let's remove these assertions because they can never fail. replace them with metadata assertions!

Reply:
This is a generic function that is used in several tests (including unit tests) and the list of options is a parameter. We also already test the metadata there. See: 

https://github.com/DataDog/integrations-core/blob/3a56e9ed3617e86e963ce0b464bdf8858db8d7bb/gitlab/tests/test_e2e.py#L14

I just pushed https://github.com/DataDog/integrations-core/pull/14716/commits/4524bc7e86a0733dcb76d470ef76808212fa18a8, let me know what you think!